app/components/spotlight/analytics/dashboard_component.html.erb

```diff
@@ -0,0 +1,51 @@
+<%= cache current_exhibit, expires_in: 1.hour do %>
+  <% if params[:start_date] || params[:end_date] %>
+    <h2><%= I18n.t("spotlight.dashboards.analytics.monthly_header_selections", start_date: dates['start_date'], end_date: dates['end_date']) %></h2>
+  <% else %>
+    <h2><%= I18n.t("spotlight.dashboards.analytics.monthly_header") %></h2>
```

Comment:
This key name `monthly_header` no longer makes sense now that we're showing a year of analytics data (and may show other ranges in the future). Maybe we should change it to something that doesn't reference a specific time, like `reporting_period_heading`